docs/databases/connections/aws-rds.md

```diff
@@ -21,6 +21,67 @@ Here's how to get connection information for databases on Amazon's RDS:
+
+To set up IAM authentication:
+
+1. [In AWS, enable IAM authentication on your RDS instance](#in-aws-enable-iam-authentication-on-your-rds-instance)
+2. [In AWS, set up an IAM policy](#in-aws-set-up-an-iam-policy)
+3. [In your database, create a database user](#in-your-database-create-a-database-user)
+4. [In your Metabase environment, configure AWS credentials](#in-your-metabase-environment-configure-aws-credentials)
+5. [In Metabase, select IAM authentication](#in-metabase-select-iam-authentication)
+6. [In Metabase, configure SSL](#in-metabase-configure-ssl)
+
+### In AWS, enable IAM authentication on your RDS instance
+
+[Enable IAM authentication on your RDS instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.Enabling.html) in the AWS console.
+
+### In AWS, set up an IAM policy
+
+Add a policy with the `rds-db:connect` action. The policy resource must specify the [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html) of your database user in the format:
+
+```
+arn:aws:rds-db:region:account-id:dbuser:DbiResourceId/db-user-name
+```
+
+Note, when entering the username _in Metabase_, you'd just enter your `db-user-name`, not the full ARN.
+
+See [Creating IAM policy for IAM database access](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.IAMPolicy.html).
+
+### In your database, create a database user
+
+Create the database user with IAM authentication enabled. The database username must match exactly (case-sensitive) with the `db-user-name` portion of your IAM policy [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html).
+
+**PostgreSQL:**
+
+```sql
+CREATE USER your_username;
+GRANT rds_iam TO your_username;
+```
+
+**MySQL:**
+
+```sql
+CREATE USER 'your_username'@'%' IDENTIFIED WITH AWSAuthenticationPlugin AS 'RDS';
+```
+
+See [Setting up for IAM database authentication](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.DBAccounts.html).
+
+### In your Metabase environment, configure AWS credentials
+
+Authentication credentials must be available via one of the methods supported by the [AWS SDK credentials chain](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials-chain.html), typically either:
+
+- Environment variables (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`).
+- AWS credentials file (`.aws/credentials`), automatically available if running in [Elastic Container Service (ECS)](https://docs.aws.amazon.com/ecs/).
+
+### In Metabase, select IAM authentication
+
+When adding or editing a database connection in Metabase, click **Use an authentication provider** and select **IAM Authentication**.
+
+### In Metabase, configure SSL
+
+Use a secure connection (SSL). Set the SSL Mode to **require**. See [PostgreSQL SSL options](./postgresql.md#ssl-mode) or [MySQL SSL options](./mysql.md#use-a-secure-connection-ssl).
```

Comment:
>Set the SSL Mode to **require**.

this is not correct.
for mysql *specifically* (this is a requirement of the mariadb driver we use), `sslMode` must be `VERIFY_CA`, but it is assumed implicitely _unless manually_ set otherwise, in which case we'll throw an error unless it is `VERIFY_CA`:

<img width="1101" height="912" alt="Image" src="https://github.com/user-attachments/assets/3e230a3a-eedc-421e-a81e-75077a9003db" />